### PR TITLE
Issue14/set up models

### PR DESCRIPTION
**Pull request by tomyang729**
我先把code changes push了，sql scripts 分在下一个PR里吧

### CODE REVIEW

File: play/src/main/scala/com/managementapp/database/dao/QuizItemsDAO.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.google.inject.Inject
+import com.managementapp.database.models.QuizItems
+import com.managementapp.services.common.ManagementAppDatabase
+import play.api.db.slick.DatabaseConfigProvider
+import slick.driver.MySQLDriver.api._
+
+import scala.concurrent.Future
+
+class QuizItemsTable(tag: Tag) extends Table[QuizItems](tag, "QUIZ_ITEMS") {
```
Comment (omzmarlon) [inline play/src/main/scala/com/managementapp/database/dao/QuizItemsDAO.scala:11]: Do we need foreign keys in this table to indicate which QuizSet this QuizItem belongs to?

Comment (tomyang729) [reply play/src/main/scala/com/managementapp/database/dao/QuizItemsDAO.scala:11]: Not really. This table only stores the info about each single quiz item. The relations between Quizes and these quiz items are stored in a separate table. See 'ItemsComposeQuizes'

Comment (omzmarlon) [reply play/src/main/scala/com/managementapp/database/dao/QuizItemsDAO.scala:11]: Ahh I see

**Review by omzmarlon — APPROVED** [review]
Great work!

**Pull request merged**